app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -409,7 +409,12 @@ function AcuantCapture(
         size: nextValue.size,
         failedImageResubmission: hasFailed,
       });
-      trackEvent(`IdV: ${name} image added`, analyticsPayload);
+
+      if (name === 'selfie') {
+        trackEvent('idv_selfie_image_file_uploaded', analyticsPayload);
```

Comment:
Should we follow standard event name pattern like 'IdV: selfie image file uploaded' etc.

Reply:
Or it's the new name standard:

can be written with ternary operator like: 

```typescript
trackEvent(name ==='selfie' ? 'idv_selfie_image_file_uploaded' : `IdV: ${name} image added`, analyticsPayload)
```

Reply:
Correct, [it is the new naming standard](https://github.com/18F/identity-idp/pull/9616/commits/3d4b4d77ddad9f1dd030463e11d61fa0a518dc9f).

Thank you for the suggestion, I've just pushed up a commit making that change.